static/js/components/ProgramCourseList.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React from 'react';
+import ProgramCourse from './ProgramCourse';
+
+
+export default class ProgramCourseList extends React.Component {
+  props: {
+    courses: Array<Object>,
```

Comment:
It would be good to make `Object` more specific, maybe with a `ProgramPageCourse` type or something like that